modules/sd-planning-considerations.adoc

```diff
@@ -56,12 +56,11 @@ If you change the number of compute nodes after installation, the control plane
 
 Post-installation scaling requirements for control plane and infrastructure nodes are assessed on a case-by-case basis. Node resource consumption and received alerts are taken into consideration.
 
-.Rules for control plane node resizing alerts
+.Rules for control plane node resizing alert
```

Comment:
Is there only one alert now? This should either be reverted to plural (for a general statement) or, if there is now only one possible alert, it needs an article ("the [...] alert").

Reply:
There is only one alert, though I think whether there are one or many alerts is an implementation detail that someone reading the docs doesn't need to care about.

Long way of saying: I'm ok with both approaches (adjusting the header/lead-in sentence to be singular or changing it back to match for consistency) :D

Reply:
Thanks for the context, @mjlshen! 

I think reverting makes the most sense, since we tend to talk in general statements in docs. 

If we want to be extra specific instead, I would add "the" to the heading for grammar/style, and also make the lead-in say "The resizing alert" instead of "A resizing alert".